code/modules/flufftext/Hallucination.dm

```diff
@@ -157,11 +158,15 @@ GLOBAL_LIST_INIT(hallucinations_major, list(
 	cost = 25
 
 /datum/hallucination/fake_flood/New(mob/living/carbon/T, forced = TRUE)
+	set waitfor = FALSE
```

Comment:
all you needed was /datum/hallucination, unset all these others please

Reply:
It was already set, and it didn't work on testing

Reply:
Maybe that's a quirk of New then. Did you test that these ones work?

Reply:
Yeah, intead of one waiting for each other they ran concurrently. Are you sure it's inherited? It feels like it's not meant to, especially if ..() is not used

Reply:
@Cyberboss